Automattic/jetpack
# Update analytics events for JITMs

#### Changes proposed in this Pull Request:

* This PR adds a `jitm_message_path` property to JITM related analytics events. 
* This allows for some differentiation for a message that may show in multiple locations.

#### Jetpack product discussion


pbNhbs-1vV-p2#comment-5193

#### Does this pull request change what data or activity we track or use?


Yes, this PR adds a new `jitm_message_path` prop to `jitm_view_client` and `nudge_click` analytics events.

#### Testing instructions:






* Apply this PR to your Jetpack test site
* Make sure you build the following packages: `jetpack build packages/jitm` `jetpack build packages/tracking`
* Build the Jetpack plugin: `jetpack build plugins/jetpack` 
* Navigate to the Jetpack dashboard of your test site. If you have not dismissed it already in the past, you should see a JITM message show at the top of the page about the Jetpack Mobile App:
![Screen Shot 2021-12-22 at 4 10 41 PM](https://user-images.githubusercontent.com/18016357/147156869-96e7391a-febd-4681-b39b-e77827eebe8d.png)

* Click on the "Download for free CTA button"
* Wait a few minutes and confirm that Tracks has received the events for `jitm_view_client` and `nudge_click` and that both have the `jitm_message_path` prop set.
